src/main/java/stream/flarebot/flarebot/util/votes/VoteUtil.java

```diff
@@ -0,0 +1,95 @@
+package stream.flarebot.flarebot.util.votes;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Guild;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import stream.flarebot.flarebot.scheduler.FlareBotTask;
+import stream.flarebot.flarebot.scheduler.Scheduler;
+import stream.flarebot.flarebot.util.ColorUtils;
+import stream.flarebot.flarebot.util.MessageUtils;
+import stream.flarebot.flarebot.util.buttons.ButtonUtil;
+import stream.flarebot.flarebot.util.objects.ButtonGroup;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class VoteUtil {
+
+    private static Map<String, VoteGroup> groupMap = new ConcurrentHashMap<>();
+    private static Map<String, VoteGroup.VoteRunnable> runnableMap = new ConcurrentHashMap<>();
+
+    public static void sendVoteMessage(UUID id, VoteGroup.VoteRunnable voteRunnable, VoteGroup group, long timeout, TextChannel channel, User user, ButtonGroup.Button... optionalButtons) {
+        EmbedBuilder votesEmbed = new EmbedBuilder();
+        votesEmbed.setDescription("Vote to " + group.getMessageDesc());
+        votesEmbed.addField("Yes Votes", "0", true);
+        votesEmbed.addField("No Votes", "0", true);
+        String messageDesc = group.getMessageDesc();
+        votesEmbed.setColor(ColorUtils.FLAREBOT_BLUE);
+        group.setVotesEmbed(votesEmbed);
+        ButtonGroup buttonGroup = new ButtonGroup(user.getIdLong());
+
+        groupMap.put(id + channel.getGuild().getId(), group);
```

Comment:
I already disapprove of the id system, at least use long id... save 8 bytes at least

Reply:
If you can come up with a better one be my guest